general/1_terminal/Mykola-Tykhomyrov/Home work 1/isWeekend(task_3).sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+today=$(date '+%a')
+if [ $today == 'Сб' ] || [ $today == 'Вс' ] || [ $today == 'Sat' ] || [ $today == 'Sun' ]
```

Comment:
Let's make it more portable.

Not all machines have Russian or English languages. Don't use string comparison in your final solution.